Example/src/TagInputExample.js

```diff
@@ -16,6 +16,8 @@ export default class TagInputExample extends Component {
     });
   };
 
+  labelExtractor = (tag) => tag;
```

Comment:
This is probably the most controversial change. `labelExtractor` replaces `labelKey`, and is now a required param. I did this so we could strongly type the item type with generics. There's no way to tell Flow "here's a default labelExtractor, but it only works when T is a string!"

`FlatList`/`VirtualizedList` has the same sort of problem, and they solved it with two components: `VirtualizedList`, which requires the extractor parameter, and `FlatList`, which wraps `VirtualizedList` and exposes a simpler interface. We could do that here too, but it's probably overkill.